test/test_linalg.py

```diff
@@ -8788,6 +8788,7 @@ def test_preferred_blas_library(self):
         self.assertEqual(out1, out2)
         self.assertEqual(out_ref, out2.cpu())
 
+    @onlyCUDA
     @skipCUDAIfNotRocm
     @unittest.skipIf(not blaslt_supported_device(), "blasLt not supported on current device")
```

Comment:
![image](https://github.com/user-attachments/assets/ce46142c-e1b7-4531-b689-ffa3cac45386)

this line should do the trick and exclude CPU. How come it isn't in your case?

Reply:
This line only checks if CUDA is available and returns true for NV-backend, it does not exclude cpu during runtime